src/olympia/devhub/templates/devhub/new-landing/components/my-addons.html

```diff
@@ -89,9 +89,9 @@
 
             <span class="DevHub-MyAddons-item-modified">
               {% if addon.last_updated %}
-                {{ _('Last Updated:') }}
+                <span>{{ _('Last Updated:') }}</span>
                 {# L10n: {0} is a date. dennis-ignore: E201,E202,W202 #}
```

Comment:
The comment is about the translatable datetime format string which is also the reason why I didn't wrap the whole string in a `trans` block.

Not sure what you mean by placeholder in this context tbh.